pkg/storageops/aws/aws.go

```diff
@@ -428,47 +414,49 @@ func (s *ec2Ops) Enumerate(
 				}
 			}
 			if !found {
-				s.addResource(sets, vol, SetIdentifierNone)
+				s.addResource(sets, vol, storageops.SetIdentifierNone)
 			}
 		}
 	}
+
 	return sets, nil
 }
 
 func (s *ec2Ops) Create(
-	v *ec2.Volume,
+	v interface{},
 	labels map[string]string,
-) (*ec2.Volume, error) {
+) (interface{}, error) {
+	vol := v.(*ec2.Volume)
```

Comment:
safe-guard against go panics.
```
vol, ok := v.(*ec2.Volume)
if !ok {
    return nil, err
}
```
Do the same at all places where you are type casting.